src/main/java/loci/common/S3Handle.java

```diff
@@ -351,12 +347,7 @@ protected void downloadObject(Path destination) throws HandleException, IOExcept
       Files.createDirectories(destination.getParent());
       s3Client.getObject(bucket, path, destination.toString());
     }
-    catch (
-      IOException |
-      InvalidKeyException |
-      MinioException |
-      NoSuchAlgorithmException |
-      XmlPullParserException e) {
+    catch (S3ClientServiceException e) {
```

Comment:
Does it still make sense to catch ```IOException``` here?  This would change what is thrown when ```Files.createDirectories``` throws ```IOException````.

Reply:
Good point. I think it's better to be consistent with the other methods which allow `IOException` to propagate. To make it clearer I've moved `Files.createDirectories` outside the try-catch.